django/db/migrations/loader.py

```diff
@@ -243,6 +249,38 @@ def build_graph(self):
             # Mark the replacement as applied if all its replaced ones are
             if all(applied_statuses):
                 self.applied_migrations.add(key)
+        # Map of inapplicable migrations rerouted to normal migrations.
+        # Anything that depends on an inapplicable migration can be rerouted
+        # to depend on applicable migrations via this map.
+        rerouted_dependencies = {}
```

Comment:
I'm not sure if "routing" is the best word choice. I didn't see this terminology used elsewhere in migrations.